proto/viam/app/v1/billing.proto

```diff
@@ -80,35 +94,83 @@ message GetCurrentMonthUsageSummaryResponse {
   double total_usage_without_discount = 11; // returns amt without any discounts applied
 }
 
+message GetCurrentMonthUsageRequest {
+  string org_id = 1;
+}
+
+message GetCurrentMonthUsageResponse {
+  google.protobuf.Timestamp start_date = 1;
+  google.protobuf.Timestamp end_date = 2;
+  double cloud_storage_usage_cost = 3;
+  double data_upload_usage_cost = 4;
+  double data_egres_usage_cost = 5;
+  double remote_control_usage_cost = 6;
+  double standard_compute_usage_cost = 7;
+  double discount_amount = 8;
+  double total_usage_with_discount = 9;
+  double total_usage_without_discount = 10;
+}
+
+// TODO(APP-1865) may want to remove
 message GetUnpaidBalanceRequest {
   string org_id = 1;
 }
 
+// TODO(APP-1865) may want to remove
 message GetUnpaidBalanceResponse {
   double unpaid_balance = 1;
   google.protobuf.Timestamp unpaid_balance_due_date = 2;
 }
 
+// TODO(APP-1865) may want to remove
 message GetInvoiceHistoryRequest {
   string org_id = 1;
 }
 
+// TODO(APP-1865) may want to remove
 message GetInvoiceHistoryResponse {
   repeated InvoiceSummary invoices = 1;
 }
 
+// TODO(APP-1865) may want to remove
 message GetItemizedInvoiceRequest {
   string id = 1;
 }
 
+// TODO(APP-1865) may want to remove
 message GetItemizedInvoiceResponse {
   Invoice invoice = 1;
 }
 
+message GetBillingInformationRequest {
+  string org_id = 1;
+}
+
+message GetBillingInformationResponse {
```

Comment:
nit: maybe OrgBillingInfo? This entire page is BillingInfo really so having Org as a prefix makes it clear its more admin-related details.

Reply:
Agreed. Changed.